web/themes/custom/pineapple/sass/layouts/nodes/_article.scss

```diff
@@ -1,48 +1,7 @@
-.article--teaser {
-  display: flex;
-  flex-direction: column;
-  background-color: $light-gray;
-  width: 100%;
-  height: 100%;
-  overflow: hidden;
-
-  @include breakpoint($desk) {
-    width: 75%;
-  }
-  @include breakpoint($palm) {
-    flex-direction: row;
-  }
-
-  .teaser-content {
-    padding: $unit * 4;
-    padding-top: 0;
-    width: 100%;
-    height: auto;
-
-    p {
-      color: $primary;
-    }
-  }
-
-  .teaser-image-container {
-    position: relative;
-    height: 100%;
-    flex-shrink: 0;
-    width: 100%;
-
-    @include breakpoint($palm) {
-      width: 33.33333%;
-    }
-
-    img {
-      height: 100%;
-      width: 100%;
-      object-fit: cover;
-      display: none;
-
-      @include breakpoint($palm) {
-        display: block;
-      }
+.view-articles {
+  .card-container {
+    @include breakpoint($desk) {
+      width: 75%;
```

Comment:
This styling should not be in this file, because it is view specific it should be moved to `_view--articles.scss` file